RobotCode/src/main/java/frc/robot/subsystems/CdTraySubsystem.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.Ports;
+import frc.lib.core.motors.TeamSparkMAX;
+
+public class CdTraySubsystem extends SubsystemBase
+{
+	// guys i think i'm starting to get the hang of thi- oh nevermind
+	private Compressor mainCompressor;
+
+	public DoubleSolenoid cdArmLeft;
```

Comment:
Variables should be private

Reply:
You can merge both solenoid declarations into `private DoubleSolenoid cdArmLeft, cdArmRight;`